subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -547,6 +516,108 @@ int nrf_cloud_encode_config_response(struct nrf_cloud_data const *const input,
 	return 0;
 }
 
+void nrf_cloud_log_control(int log_level)
```

Comment:
does this belong?

Reply:
It's a place holder.  I added a comment.